src/renderer/components/WebPlatformPlayer/index.js

```diff
@@ -123,7 +128,12 @@ const WebPlatformPlayer = ({ manifest, onClose, inputs, config }: Props) => {
   }, [manifest.url, theme, inputs, manifest.params]);
 
   const listAccounts = useCallback(() => {
-    return accounts.map(account => serializePlatformAccount(accountToPlatformAccount(account)));
+    return accounts.map(account =>
+      accountToPlatformAccount(
+        account,
+        account.type === "TokenAccount" ? accounts.find(a => a.id === account.parentId) : undefined,
+      ),
+    );
   }, [accounts]);
 
   const listCurrencies = useCallback(() => {
```

Comment:
Should implement filtering based on params supported by the SDK method (`name`, `ticker`, `type`)
cf. **[here](https://github.com/LedgerHQ/live-app-sdk/pull/36/files#diff-ccf0ddb1fb1db2594f06db30d2c40af22dfc0bf0744a822a71bf4bd8d0880aa5R305-R318)**